index.html

```diff
@@ -121,8 +125,8 @@ <h3>Itachi Uchiha</h3>
               created by Masashi Kishimoto.
             </span>
           </div>
-        </div>
-        <div class="speaker-card">
+        </div> -->
+        <!-- <div class="speaker-card">
```

Comment:
Kindly, I suggest you remove the commented code, it's not a good practice, it will also make your code look cleaner and more professional.

Reply:
Done. Thank you for your reviews